js/game.js

```diff
@@ -57,6 +57,8 @@ class Game {
     }
   }
 
+  // todo --> this method is huge, and it also calls functions from main.js. I'd appreciate any feedback
+  //  on ways to slim it down and remove those calls because I think we're not supposed to do that.
   slap(player) {
```

Comment:
Honestly this looks pretty good. When I did this project, this function was messy and long also, I think it's just the nature of this project.